plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -295,32 +345,6 @@ class NewServiceFormModal extends Component {
       serviceReviewActive
     } = this.state;
 
-    let errorsMap = new Map();
-    if (this.props.errors) {
-      let message = this.props.errors.message;
-
-      if (this.shouldForceSubmit()) {
-        message = `App is currently locked by one or more deployments.
-         Press the button again to forcefully change and deploy the
-         new configuration.`;
-      }
```

Comment:
Hey @jfurrow , this is replaced by the `SERVICE_DEPLOYING` detected in `MarathonErrorUtil.getErrorType`  . The actual error message is added when the error message translation is introduced in a later PR. Till then, the default marathon error will be shown.